rdkit/Chem/PandasTools.py

```diff
@@ -379,50 +188,48 @@ def _molge(x, y):
   else:
     return False
 
-def PrintAsBase64PNGString(x, renderer=None):
-  '''returns the molecules as base64 encoded PNG image
-    '''
+def PrintAsBase64PNGString(x):
```

Comment:
Do you want to document this deprecation in the ReleaseNotes.md so that we can remove it in the next release?

Reply:
Done.